config/pipelines/high_throughput_heron_384_tailed.yml

```diff
@@ -0,0 +1,26 @@
+---
+Heron-384 Tailed A: # Heron 384-well pipeline specific to PCR 1 plate
+  filters:
+    request_type_key: limber_heron_lthr
```

Comment:
Why do we need a new request type? Just noticing the current 96 and 384 pipelines have the same, and have forgotten the implications of request types vs library types etc.

Also, could it be called limber_heron_tailed, or is that too long? More human-readable.

Reply:
Best to keep request types for different pipelines (processes) distinct, plus confusing for user if more than one suggested action visible in Limber. Library types are mainly for NPG to tell libraries apart for analysis, but we can also use them in Limber to further determine suggested action.

Reply:
Thanks. The existing 96 and 384 pipelines use the same request type, so I guess we avoid the "more than one suggested action visible in Limber" problem by also using the library type to distinguish between them?

So we could theoretically do the same here, but I guess it makes sense to have a new request type because they have developed a new process, whereas the existing 96 and 384 pipelines basically follow the same process (apart from different plate sizes).

Reply:
Yeah, we don't need one, but Lesley specifically requested one, mostly to try and avoid the SSRs accidentally mixing up library types (by pushing the main decision back to the submission template). Initially the requirements actually specified distinct request types for the two plate sizes, but we walked that one back to maintain consistency.

But @KatyTaylor is right, and the library_type would stop the users from seeing the other suggested actions. Its mostly the SSR this will affect. (It mean a separate submission template for the tailed, rather then narrowing down a shared template with specific library type options)